### DISCUSSION TITLE
Web GUI access

### DISCUSSION THREAD

**Discussion by jhernandez-sun** [Security-Onion-Solutions/securityonion#7124] [Unsupported Versions]
Hello

I'm in need of help.

I have installed SO 2.3, so good so far. After I few minutes after installation and testing, suddenly I was unable to access the web GUI. At first I thought I had messed up some server configuration, but then it hit me, it's the firewall. When I stop the service, I regain access for the web gui, unfortunately, after some time, I lose access to the gui. As long I start and stop the firewall service I can access the gui.

Is there some documentation or some insight in how to solve this issue?

Any help greatly appreciated.

**Comment by Acewiza** [comment]
Are you the firewall operator?

**Reply by jhernandez-sun** [reply]
The firewall in question is the CentOS server firewall, where Security Onion is located.

**Comment by dougburks** [comment]
Have you tried running `so-allow` to allow your traffic through the host firewall?
https://docs.securityonion.net/en/2.3/so-allow.html#so-allow

**Reply by jhernandez-sun** [reply]
Hello

Yes, I have tried the **so-allow** but this only reports that the address already exists.
The firewall I'm talking about, is the firewall present in the CentOS server, where Security Onion is hosted.

I'm using the command **sudo systemctl start firewalld** and **sudo systemctl stop firewalld**

**Reply by dougburks** [reply]
Yes, `so-allow` allows traffic through the iptables firewall built into CentOS on your Security Onion box. If you've already added the correct `so-allow` entry, then you shouldn't need to stop the firewall. If you have Wazuh enabled, it's possible that Wazuh Active Response is blocking your IP address in the firewall, although `so-allow` should add your IP address to the Wazuh safe list to prevent that. 

Do you have Wazuh enabled?

If so, do you see your analyst IP address listed when running the following command?
```
sudo grep white_list /nsm/wazuh/etc/ossec.conf
```

**Reply by jhernandez-sun** [reply]
Yes, Wazuh enabled, and yes, my analyst IP is white-listed. It's white-listed twice, one created by SETUP, the other created by SO-ALLOW

**Reply by dougburks** [reply]
If your IP is whitelisted properly, then Wazuh shouldn't block it in the firewall. You could investigate that further by checking `/nsm/wazuh/logs/` to see if there is any evidence that Wazuh is engaging active response against your IP.
